#
# This patch fixes this bug: https://github.com/Hacker0x01/react-datepicker/issues/3834
# suggested in this comment: https://github.com/Hacker0x01/react-datepicker/issues/3834#issuecomment-1562485580
#
# Bug if this is not patched: webpack will load the commonjs version of the lib instead of the esm version. React will throw this error:
# Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.
#

diff --git a/node_modules/react-datepicker/package.json b/node_modules/react-datepicker/package.json
index ce0f5d7..a7a5c7c 100644
--- a/node_modules/react-datepicker/package.json
+++ b/node_modules/react-datepicker/package.json
@@ -6,7 +6,6 @@
   "license": "MIT",
   "homepage": "https://github.com/Hacker0x01/react-datepicker",
   "main": "dist/index.js",
-  "browser": "dist/react-datepicker.min.js",
   "module": "dist/es/index.js",
   "unpkg": "dist/react-datepicker.min.js",
   "style": "dist/react-datepicker.min.css",
